pyomo/devel/example/plugins/ex_plugin.py

```diff
@@ -13,8 +13,7 @@
 
 
 @TransformationFactory.register(
-    'contrib.example.xfrm',
-    doc="An example of a transformation in a pyomo.contrib package",
+    'devel.example.xfrm', doc="An example of a transformation in a pyomo.devel package"
 )
 class Xfrm_PyomoTransformation(Transformation):
     def __init__(self):
```

Comment:
Perhaps while we're making this change, we should adjust naming here? I'd suggest `devel.example.transform` with a class name of just `ExamplePyomoTransformation` or something. I also think it would be good to not have a `plugins` directory in `example` and just flatten it out like we wish we had so many other places...